### DISCUSSION TITLE
使用h5标签在小程序端没有显示

### DISCUSSION THREAD

**Discussion by wsq-wushiqi** [NervJS/taro#10680] [Q & A]
// package.json
"dependencies": {
    "@babel/runtime": "^7.7.7",
    "@tarojs/components": "3.3.14",
    "@tarojs/plugin-html": "^3.3.13",
    "@tarojs/runtime": "3.3.14",
    "@tarojs/taro": "3.3.14",
    "vue": "^2.5.0",
    "vue-template-compiler": "^2.5.0"
  },
// config/index.js
plugins: ['@tarojs/plugin-html'],

// index.vue
<template>
  <div>h5组件</div>
</template>

以上，插件@tarojs/plugin-html装了，config配置了，使用div标签在微信小程序端显示的是空白

**Comment by Chen-jj** [comment]
@wsq-wushiqi 参考一下 https://github.com/NervJS/taro-vant，应该是可以跑起来的

**Reply by wsq-wushiqi** [reply]
这个是可以跑起来。但是自己新建的项目还是不行，除了文档说的安装插件和引用插件，是否还有其他注意事项呢？

**Reply by jojo-1995** [reply]
我也是，升级到了3.3.9，按照官方配置了，直接写原生h5标签，空白白

**Comment by Chen-jj** [comment]
@wsq-wushiqi @jojo-1995 按文档说的配置即可，如果还有问题，需要提供一下 demo 了，猜不到出了什么问题

**Comment by wsq-wushiqi** [comment]
@Chen-jj @jojo-1995 排查出来了，1.@tarojs/plugin-html要安装到开发依赖；2.插件版本要与@tarojs/taro版本号一致；3.如果还不行的话，就把版本号前面的插入符号^去掉
